### DISCUSSION TITLE
pycharm 能调试启动吗？

### DISCUSSION THREAD

**Discussion by water-boil-meat** [nonebot/discussions#164] [问答] [answered]
菜鸡 python新手找不到 main方法

**Comment by MingxuanGame** [comment, marked as answer]
你可以使用运行 `bot.py` 的方式

![image](https://user-images.githubusercontent.com/68982190/172817154-7a4bbed6-6f10-4eb6-8e4d-711527d19812.png)

或者运行模块 `nb_cli` 然后添加形参 `run`

![image](https://user-images.githubusercontent.com/68982190/172817244-92ab6385-ec2e-4108-9ccc-e7daa8d3a434.png)

**Reply by MingxuanGame** [reply]
然后只需要在你需要暂停的位置打断点（CTRL+F8）即可

**Reply by water-boil-meat** [reply]
nb_cli 模块在哪？
我在项目目录没看到。。

**Comment by synodriver** [comment]
直接断点调试bot.py即可
